.github/workflows/channels-ci.yml

```diff
@@ -188,7 +188,6 @@ jobs:
           make node_modules
           # make channels/e2e/playwright/node_modules
       - name: ci/build
-        working-directory: webapp/channels
         run: |
           npm run build
```

Comment:
To double check, are we missing the `make dist` step that puts the production assets from Playbooks and Boards into the `dist/` folder?

Reply:
That's not actually needed here. This check is only to confirm that the individual packages build. We're not producing an artifact or putting that together here. That's part of the server CI pipeline that generates the whole bundle

Reply:
Hmm, but why would we need to `npm run build` more than once? Isn't that a waste? I'm trying to stop thinking of a "server CI" and a "webapp CI", but just a general CI for the entire monorepo.

Reply:
We could definitely consider changing that going forward, but that can be a followup.

I think treating them somewhat independently makes sense though for things that are built independently to avoid having to run the entire CI pipeline when something changes that won't affect the others. Maybe we decide to merge these "build" checks, but for the linter and unit tests, I'd like to make it so that those are only run when that part of the code has actually changed. I'd rather not have to wait for all the server unit tests to complete when I submit my web-app-only PR, and I'd expect people on the server side would feel the same way

Reply:
In general, I agree! Given that every PR -- even if not webapp related -- needs to build the webapp in order to product an image, it seems like we can always guarantee the build step here gets run as part of the overall build process.

Love the idea of skipping lints when we know nothing in that branch has changed though!